schema.json

```diff
@@ -519,7 +519,7 @@
             "$ref": "#/definitions/uriOrArrayOfUris"
         },
         "version": {
-            "pattern": "^[\\w\\.\\-_]+$",
+            "pattern": "^[\\w\\.\\+\\-_]+$",
```

Comment:
```suggestion
            "pattern": "^[\\w.+-_]+$",
```

Reply:
Why not `"^[\\w-.+_]+$"` ?

Reply:
I do not know precedence of evaluating in powershell. Because `-` is control sequence character in character classes `+-.` mean "Everything between plus and dot" == "Everything between 43 and 46 character index"

Reply:
It should be `"^[\\w.+_-]+$"`, i.e., putting `-` as latest character will get literal `-`. (https://regex101.com/r/ekwiw7/1)

Reply:
I think it could be even after `\w`, but that depends on implementation. You could put it on the end, but i would prefer to escape it as it will not be problem in any of implementations.

Reply:
Yes, `^[\\w-.+_]+$` should work, too. PowerShell (or .NET) follows PCRE flavor and `-` follows `\w` means `-`. (https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expressions#how-regular-expressions-work)